docs/source/tagset/elemento-back.rst

```diff
@@ -3,13 +3,25 @@
 <back>
 ======
 
-Aparece em
+Aparece em:
+
   :ref:`elemento-article`
 
-Ocorre
+Ocorre:
+
   Zero ou uma vez
 
-O ``<back>`` é a parte final do :term:`documento` que compreende lista de referências e demais
-dados referentes a pesquisa como: notas de rodapé, agradecimentos, apêndice,
-material suplementar, anexos e glossário.
- 
+Parte final do :term:`documento` que compreende:
+
+* lista de referências;
+* notas de rodapé;
+* agradecimentos
+* apêndice(s)
+* material suplementar
+* anexos; e
+* glossário.
+
+Exemplo:
```

Comment:
@gustavofonseca , vejo o ``<back>`` como um conteiner vazio onde podem ser colocados diversos elementos de diversos tipos e funções. Não sei se precisa explicar muito. De repente, com um exemplo pode-se fazer o usuário entender que é só uma 'seção' que separa o artigo de forma estrutural (início, meio, fim).